.php_cs.dist

```diff
@@ -9,7 +9,7 @@ return PhpCsFixer\Config::create()
         'binary_operator_spaces' => true,
         'blank_line_before_return' => true,
         'cast_spaces' => true,
-        'concat_space' => ['spacing' => 'one'],
+        'concat_space' => ['spacing' => 'none'],
```

Comment:
I'm actually not a massive fan of this particular change - just a personal preference. In fact, this config basically comes from [Flysystem](https://github.com/thephpleague/flysystem/blob/1.x/.php_cs.dist)

Reply:
I think concatenation must be concatenated :laughing:

Reply:
Laravel uses `'concat_space' => ['spacing' => 'none'],`

You can check their preset: https://docs.styleci.io/presets#laravel (aka concat_without_spaces)

Reply:
I respectfully don’t care what Laravel uses in the context of this project.

Reply:
This is my preferred style and what other league packages do.